src/main/java/project_16x16/projectiles/MagicProjectile.java

```diff
@@ -109,8 +109,13 @@ public void hit(CollidableObject collision) {
 	private void setParticleAnimation(SideScroller a) {
 		particleAnimation = new ArrayList<PImage>();
 		PImage image = Tileset.getTile("MAGIC_SOURCE");
-		float angle = PApplet.radians((float) (Math.random() * 360));
-		particleAnimation.add(Util.resizeImage(Util.rotateImage(image.copy(), angle), 0.25f));
+		float scale = 0.12f;
+		float angle = PApplet.radians(11);
+		while(scale > 0.025f) {
+			particleAnimation.add(Util.resizeImage(Util.resizeImage(Util.rotateImage(image.copy(), angle), scale), 4));
+			angle += PApplet.radians(11);
+			scale -= Math.random() * 0.03f;
+		}
```

Comment:
I removed this block before because it doesn't actually run.

Reply:
it makes the particle smaller and rotate?

Reply:
Oh my bad -- looked closer and it does run --  I thought it was scale > 0.25.